### ISSUE TITLE
Creating/deleting dir triggers file change events

### ISSUE THREAD

**Issue by caikan** [floatdrop/gulp-watch#236]
```
gulpWatch(['src/**'], function(vinyl) {
    console.log(vinyl.event, vinyl.relative);
});
```

Windows 7.
When I create a directory named [a], the files whose name beginning with [a] will trigger a change event.

**Comment by UltCombo** [comment]
Maybe related to #231? I haven't got time to triage it yet.

**Comment by UltCombo** [comment]
Confirmed bug, managed to reproduce it in Windows 10.

Most likely a Chokidar bug, will have to investigate further.

**Comment by UltCombo** [comment]
Upstream bug reference: paulmillr/chokidar#499

**Comment by UltCombo** [comment]
This may be fixed in Node v7.1.0 as it uses libuv 1.10.0 which includes a fix for this issue. Can someone test on a Windows machine to confirm?

**Comment by UltCombo** [comment]
I've tested on a Windows machine now, with Node 6.9.1 (reproduces the issue) and Node 7.1.0 (works as expected).

The bug was in libuv, part of the Node.js core. It is [fixed](https://github.com/libuv/libuv/commit/77c8abae33351acc3a845aebb61f485c94d61cdf) as of libuv 1.10.0, which ships with Node 7.1.0.
